packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: System Log events for rate limits
+category: rate limits
+---
+
+# System Log events for rate limits
+
+Okta offers three standard System Log event types that you may encounter when Okta is enforcing [rate limits](/docs/reference/rate-limits/) on specific operations.
+
+* `system.operation.rate_limit.violation`<br>
+This event type is sent once per rate limiting period when a request is rejected for exceeding a rate limit. For example, if the rate limit that was exceeded has a reset period of one minute, then one event of this type is emitted during that period for the applicable scope.
+
+* `system.operation.rate_limit.warning`<br>
+This event type may be sent once per rate limiting period as a warning that some significant portion of your rate limit has already been used within a period. For example, you may receive a warning that you have reached 60% of your rate limit for an endpoint within a rate limiting period.
+
+* `system.operation.rate_limit.notification`<br>
+This event type can provide additional information about rate limiting decisions. For example, a violation event would have been emitted if you had a different configuration.
```

Comment:
"For example, a violation event would have been emitted if you had a different configuration." ->
"For example, this event may indicate that a violation event would have been emitted for just a particular client rather than for a broader scope if you had chosen a different configuration."